src/create-element.js

```diff
@@ -79,7 +79,7 @@ export function createVNode(type, props, key, ref, original) {
 		_original: original
 	};
 
-	if (original == null) vnode._original = vnode;
+	if (original == null) vnode._original = {};
```

Comment:
Could we use `EMPTY_OBJ` here?

Reply:
no then every vnode would be equal and bail out of every render